### PR TITLE
Make separate docker-compose environments

### PR DESCRIPTION
**Pull request by freeatnet**
This is a devops change. It's made on top of #22, see e6bc7ad for changes.

TL;DR: add `COMPOSE_FILE=docker-compose.yml:docker-compose.development.yml` to your `.env` to keep the current development environment behaviour.

We need a stable environment on staging/production, where we bake the source code into the image, and the code doesn't change if the working copy code changes. This is achieved by removing the directive of `docker-compose.yml` that mounts code into the containers. Because we still want to keep the old behaviour in development, we actually extract the code-mounting directive into `docker-compose.development.yml`, which is applied on top of `docker-compose.yml` using `COMPOSE_FILE` environment variable.

This PR:
* Moves development-environment config out of `docker-compose.yml` and into `docker-compose.development.yml`
* Adds some pointers on how that setup works.

### CODE REVIEW

**Review by JonathonDunford — CHANGES_REQUESTED** [review]
Can we add some documentation on this in the README? Even if it's a couple sentences.

**Comment by freeatnet** [comment]
@JonathonDunford Added a note about uncommenting the `COMPOSE_FILE` line in `.env` to README.

**Comment by JonathonDunford** [comment]
@freeatnet I don't see that in this PR?

**Review by JonathonDunford — APPROVED** [review]
